docs/api/cozy-client.md

```diff
@@ -2217,6 +2226,25 @@ Fetch and build an URL to open a note.
 | client | <code>object</code> | CozyClient instance |
 | file | <code>object</code> | io.cozy.file object |
 
+<a name="isShortcutCreatedOnTheRecipientCozy"></a>
+
+## isShortcutCreatedOnTheRecipientCozy
+Returns if the shortcut for the sharing is created on the recipient's cozy
+
+To know if the shortcut is created on the recipient's cozy, we need to check
+the presence of the `instance` attribute on the sharing member. (since we
+are in a cozy to cozy sharing, we have only one member per "token")
+
+If this field is setted, it means the stack knows the url of the recipient's
+cozy. And if the stack knows that, we're sure that the stack has already created
+the shortcut on the recipient's cozy.
```

Comment:
This doc is an explanation of the implementation. I am not sure this is the best information to put here; when reading the doc I would be more interested to know where this information (whether the shortcut is created on the recipient cozy) can be used.

s/setted/set

Reply:
Is it clearer now?